curations/gem/rubygems/-/rubocop.yaml

```diff
@@ -1,11 +1,15 @@
 coordinates:
-  name: rubocop
-  provider: rubygems
-  type: gem
+    type: gem
+    provider: rubygems
+    namespace: ""
```

Comment:
I'm not sure if having `""` as the namespace will cause issues since when there isn't one it usually is omitted or "-". Asking in Discord.